NerdyNomads/Text-Savvy
# Added ability to add and remove collaborators to a workspace

## What does this PR do?
- Added new endpoints:
  - Get the account with the given email
  - Get a list of workspaces that the given email is a collaborator of
- Added backend logic to add and remove collaborators to a workspace
- Moved requests to all be in the `request.js` file

### Steps to test/review
1. Add a collaborator to your workspace and save.
2. Log in to that account (or look in Mongo DB) to see if it has the workspace.
3. Delete the collaborator from the workspace. 
    - The owner can delete the collaborator OR
    - The collaborator can delete themselves

## Checklist before merging
      - Reviewers
      - Assignee
      - Labels
      - Project
      - Milestone
      - Linked Issues
